tests/databrowser.js

```diff
@@ -157,40 +156,13 @@ define([
                 });
             });
 
-            it("should add recursive ref on drag/drop", function(done) {
```

Comment:
should add some validation to assert that we don't have recursive relationships, but I don't think that's a blocker and it's reasonable to assume HQ will deliver a valid schema.